### PR TITLE
Fix watchdog modprobe return error

### PR DESCRIPTION
**Pull request by kylazhang**
Signed-off-by: Kyla Zhang <weizhan@redhat.com>

### CODE REVIEW

**Comment by kylazhang** [comment]
```
 (1/8) type_specific.io-github-autotest-libvirt.watchdog.action_shutdown.model_i6300esb.model_test: PASS (77.98 s)
 (2/8) type_specific.io-github-autotest-libvirt.watchdog.action_dump.long_domain_name.model_i6300esb.model_test: PASS (103.71 s)
 (3/8) type_specific.io-github-autotest-libvirt.watchdog.action_dump.normal_domain_name.model_i6300esb.model_test: PASS (111.67 s)
 (4/8) type_specific.io-github-autotest-libvirt.watchdog.action_none.model_i6300esb.model_test: PASS (227.81 s)
 (5/8) type_specific.io-github-autotest-libvirt.watchdog.action_poweroff.model_i6300esb.model_test: PASS (78.49 s)
 (6/8) type_specific.io-github-autotest-libvirt.watchdog.action_pause.model_i6300esb.model_test: PASS (78.87 s)
 (7/8) type_specific.io-github-autotest-libvirt.watchdog.action_reset.model_i6300esb.model_test: PASS (109.74 s)
 (8/8) type_specific.io-github-autotest-libvirt.watchdog.action_inject_nmi.model_i6300esb.model_test: PASS (77.77 s)
```

**Comment by kylazhang** [comment]
@smitterl could you please check it? Thanks

File: virttest/libvirt_xml/devices/watchdog.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def try_modprobe(self, session):
         """
         handled_types = {"ib700": "ib700wdt", "diag288": "diag288_wdt"}
         if self.model_type not in handled_types.keys():
-            return False
```
Comment (smitterl) [inline virttest/libvirt_xml/devices/watchdog.py:44]: Please, add a `logging.debug`, something like `"Unknown model type. Skipping watchdog modeprobe for %s." % self.model_type`.

**Review by smitterl — CHANGES_REQUESTED** [review]

File: virttest/libvirt_xml/devices/watchdog.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def try_modprobe(self, session):
         """
```
Comment (smitterl) [inline virttest/libvirt_xml/devices/watchdog.py:41]: Please update the comment accordingly.

**Review by smitterl — CHANGES_REQUESTED** [review]

File: virttest/libvirt_xml/devices/watchdog.py (hunk 2 above)
Comment (kylazhang) [reply virttest/libvirt_xml/devices/watchdog.py:44]: Done

File: virttest/libvirt_xml/devices/watchdog.py (hunk 1 above)
Comment (kylazhang) [reply virttest/libvirt_xml/devices/watchdog.py:44]: Done

**Review by Yingshun — APPROVED** [review]

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**